client/login/wp-login/login-links.jsx

```diff
@@ -230,6 +239,11 @@ export class LoginLinks extends React.Component {
 			return null;
 		}
 
+		// jetpack cloud needs to keep users in the flow
+		if ( isJetpackCloudOAuth2Client( this.props.oauth2Client ) ) {
```

Comment:
Here we are removing the reset your password link. 
I know its not idea but it is definitely something that users are looking for. 

Will we be adding it a back in a different PR?

Reply:
Maybe instead of removing the link we can send the user somewhere else?